api/authorization.go

```diff
@@ -14,8 +14,17 @@ import (
 )
 
 var (
-	authorizationHeaders = []string{"Authorization", "Cookie"}
-	authTimeout          = 3 * time.Second
+	authorizationHeaders = []string{"Authorization", "Cookie", "Origin"}
```

Comment:
I don't get why the `Origin` header is added here, is it getting copied by the request and sent back?

Reply:
The `Origin` header is copied from the original request into the authorization request made to the Studio API. This is here so that the API can apply the proper CORS-logic to the given request, then that's why we also started proxying all the CORS headers back from the auth request response into our real response.